particle-accelerator/src/renderer/utilities/Product.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Product: class for defining Product objects retrieved from the Particle API
+ * Abstraction for representing product families in a Particle account.
+ */
+export default class Product {
+  name
+
+  id
+
+  deviceType
+
+  constructor(name, id, platform_id) {
+    this.name = name
+    this.id = id
+    switch (platform_id) {
+      case 3:
```

Comment:
Could you please add a comment with a link to the documentation for these different platform IDs?

Reply:
Done!